doc/installation.xml

```diff
@@ -459,18 +459,21 @@ pacman -S \
     directory.
   </para>
   <para>
-    To build the documentation for the development version, simply build
-    the <literal>html</literal> or <literal>pdf</literal> targets for the
-    HTML and PDF versions of the documentation, respectively.
+    To build the documentation for the development version, simply build the
+    <literal>html</literal>, <literal>pdf</literal> or <literal>info</literal>
+    targets for the HTML, PDF and Info versions of the documentation,
+    respectively.
   </para>
 <programlisting>
 $ cmake --build . --target html
 </programlisting>
   <para>
     Building the HTML documentation requires Python 3, <literal>xmlto</literal>
-    and <literal>source-highlight</literal>. Building the PDF documentation
-    also requires <literal>xsltproc</literal>, <literal>xmllint</literal>
-    and <literal>fop</literal>.
+    and <literal>source-highlight</literal>. Building the PDF documentation also
+    requires <literal>xsltproc</literal>, <literal>xmllint</literal> and
+    <literal>fop</literal>. Building the Info documentation also requires
+    <literal>docbook2x</literal>, <literal>xmllint</literal> and
+    <literal>makeinfo</literal>.
```

Comment:
```suggestion
    and <literal>source-highlight</literal>. Building the PDF documentation also
    requires <literal>xsltproc</literal>, <literal>xmllint</literal> and
    <literal>fop</literal>. Building the Texinfo documentation also requires
    <literal>docbook2texi</literal>, <literal>xmllint</literal>
    and <literal>makeinfo</literal>.
```

This suggestion assumes that we go with `docbook2texi`, which I think we should. This command is provided by at least these two separate packages:

 - https://docbook2x.sourceforge.net/
 - https://sourceware.org/docbook-tools/

The unusual naming in Debian is very likely an attempts at avoiding naming conflicts between these two packages. With MacPorts these two cannot be installed at the same time.

Reply:
It seems that I was right about why Debian renames this program: https://bugs.gentoo.org/205703

But I was wrong in assuming that the two programs called `docbook2texi` are interchangeable. The one from docbook-tools / docbook-utils does not work here. I'm getting this:

```
FAILED: doc/igraph-docs.texi /Users/szhorvat/Repos/igraph-main/igraph/build/doc/igraph-docs.texi 
cd /Users/szhorvat/Repos/igraph-main/igraph/build/doc && /opt/local/bin/docbook2texi --encoding=utf-8//TRANSLIT --string-param output-file=igraph-docs --string-param directory-category=Libraries --string-param directory-description="A\ fast\ graph\ library\ (C)" igraph-docs-with-resolved-includes.xml
-e Usage: jw [<options>] <sgml_file>
where <options> are:
...
```

This present a dilemma as hard-coding `docbook2x-texi` will prevent the build from working on some systems, but using `docbook2texi` is also not fully reliable.

CMake's `find_program` allows specifying multiple names, so we could prefer `docbook2x-texi`, then fall back to `docbook2texi` if that is not present.

`find_program` also has a feature to validate candidates, but this feature is only available in CMake 3.25 or later.

What do you suggest @ntamas?